llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.h

```diff
@@ -47,25 +47,30 @@ class WebAssemblyTargetLowering final : public TargetLowering {
 
   enum WasmAddressSpace {
     DEFAULT = 0,
-    EXTERNREF = 1,
-    EXTERNREF_GLOBAL = 2,
-    FUNCREF = 3,
-    FUNCREF_GLOBAL = 4,
+    EXTERNREF = 10,
```

Comment:
Why did you decide to number them like this as opposed to sequentially?

Reply:
In order to not update the tests.  however let me check, i have a fresh rebased patch---